Source/DafnyTestGeneration/GenerateTestsCommand.cs

```diff
@@ -86,8 +90,8 @@ private enum Mode {
     "Print the Boogie code used during test generation.") {
     ArgumentHelpName = "filename"
   };
-  public static readonly Option<bool> DisablePrune = new("--no-prune",
-    "Disable axiom pruning that Dafny uses to speed up verification.") {
+  public static readonly Option<bool> ForcePrune = new("--force-prune",
+    "Enable axiom pruning that Dafny uses to speed up verification. This may negatively affect the quality of tests.") {
```

Comment:
Do you have a summary of why pruning negatively affects tests?

Reply:
It used to be the case that Boogie would not print the `{:uses}` attributes when saving an AST to file causing pruning to be way to aggressive if you then run Boogie on the same file. But I just discovered a [recent PR to Boogie](https://github.com/boogie-org/boogie/pull/765) that might have fixed it. I will look into it and remove the ForcePrune flag if that is the case once the newer Boogie is merged.

Reply:
Ah, I'm glad that fix might help!